jupyterlab_search_replace/tests/test_handlers.py

```diff
@@ -1,5 +1,7 @@
 import json
 import pytest
+import asyncio
+from ..search_engine import SearchEngine
 from jsonschema import validate
 from tornado.httpclient import HTTPClientError
```

Comment:
import ordering change

```suggestion
import asyncio
import json

import pytest
from jsonschema import validate
from tornado.httpclient import HTTPClientError

from ..search_engine import SearchEngine
```